test/Models/WebMapServiceCatalogItemSpec.js

```diff
@@ -35,6 +41,18 @@ describe('WebMapServiceCatalogItemViewModel', function() {
         expect(wmsItem).toBeDefined();
     });
 
+    it('add addAttributionText methods called', function() {
```

Comment:
This test and the one below are both failing for me, because globeormap is never associated with the `Terria` instance, so enabling `wmsItem` doesn't add it to that map (it adds it to the `NoViewer` instance in `terria.currentViewer`).

I think you want to remove the code above that creates a `GlobeOrMap` instance, and change the `spyOn` below to spy on `terria.currentViewer`.